### PR TITLE
Multidays select functionality

### PR DESCRIPTION
**Pull request by PatrykSzwer**
This PR adds new functionality for web component, to select multiple days at month view, and create event for them. It's necessary for https://github.com/StarcounterApps/Calendar/issues/39

### CODE REVIEW

**Comment by warpech** [comment]
Should I continue with the review? I know @PatrykSzwer you are still making changes there, right?

**Comment by PatrykSzwer** [comment]
I'll let You know when It's finished, I'm still struggling with some bugs there.

**Comment by PatrykSzwer** [comment]
Yes, now You can continue. I've pushed fix on that week view just now.

**Comment by warpech** [comment]
@un-tone could you pls review this?

**Comment by warpech** [comment]
It can be tested as part of https://github.com/StarcounterApps/Calendar/pull/57

File: sgb-fullcalendar.html [hunk 1]
```diff
@@ -173,20 +171,25 @@
...
+							var d = new Date();
+                            var diff = new moment.duration(end._d - start._d);
 
 	                        if (start.hasTime() && end.hasTime()) {
-	                            var d = new Date();
 	                            self.eventSelection = start.toJSON() + ";" + end.toJSON() + ";" + d.getTime(); // add number so the patch is sent
-	                            //jQuery(self.container).fullCalendar('unselect');
-	                        }
+                            } else if (diff.asDays() !== 1) { // diff.asDays() !== 1 for preventing unnecessary calling when invoking day view
+								start.local();
```
Comment (un-tone) [inline sgb-fullcalendar.html]: I'd recommend to replace tabs with spaces according to https://starcounter.gitbooks.io/guidelines/content/coding_style/javascript_code_formatting.html in whole file

Comment (PatrykSzwer) [inline sgb-fullcalendar.html]: Ok, done.

**Pull request merged**